src/common/options/rgw.yaml.in

```diff
@@ -3229,3 +3229,10 @@ options:
   - rgw
   flags:
   - startup
+- name: rgw_backend_store
+  type: str
+  level: advanced
+  desc: Backend store type
```

Comment:
I'm not sure that this is how we want to pick the store long term, so can we add to the description that this is experimental, so that we can easily change it to some other method if that's what we want?

Reply:
done.